srml/system/src/lib.rs

```diff
@@ -278,6 +278,13 @@ decl_module! {
 				storage::unhashed::kill(&key);
 			}
 		}
+
+		/// Kill all storage items with a key that starts with the given prefix.
+		#[weight = SimpleDispatchInfo::FixedOperational(10_000)]
```

Comment:
What should we set this to?

Reply:
cc @kianenigma

Reply:
it's entirely dependent on state, so we'd need a specialised weight type. and calculating it would take about as long as actually doing it, so kind of pointless. in any case it's root-only, so doesn't actually matter.

Reply:
https://github.com/paritytech/substrate/issues/3730